springignore.txt

```diff
@@ -6,4 +6,7 @@
 #
 /.git
 /.svn
-/recoil-lua-library
\ No newline at end of file
+/recoil-lua-library
+/lux.toml
+/.busted
+/lux.rock
```

Comment:
I can't seem to find any references online to a file called lux.rock, is this supposed to be lux.rock?

Reply:
This must've happened during setup when I was playing with `lx`, which is their shim for luarocks. Since it *can* happen under this toolchain maybe we leave it?